frameworks/hdfs/TEST_README.md

```diff
@@ -4,6 +4,7 @@ Running test.sh
 Environmental Variables
 -----------------------
 CLUSTER_URL: set this variable to use a pre-existing DC/OS cluster. For example, CLUSTER_URL=https://33.175.150.150
+TEST_TYPES: which types of tests to run: sanity, recovery. For example, TEST_TYPES=recovery
```

Comment:
Maybe mention that the default is sanity

Also the filter should also support compound statements like `TEST_TYPES="sanity or recovery"`